MachineLearning/HHMachineLearning/parameters_boosted.py

```diff
@@ -0,0 +1,398 @@
+test_cache = os.path.join(path_out,'test_cache_'+'_'.join([str(era) for era in eras])+'.pkl')
+
+# Meta config info #
+xsec_json = os.path.join(main_path,'background_{era}_xsec.json')
+event_weight_sum_json = os.path.join(main_path,'background_{era}_event_weight_sum.json')
+
+# Training resume #
+resume_model = ''
+
+# Output #
+output_batch_size = 50000
+split_name = 'tag' # 'sample' or 'tag' : criterion for output file splitting
+
+##############################  Evaluation criterion   ################################
+
+eval_criterion = "eval_error" # either val_loss or eval_error or val_acc
+
+##############################  Model callbacks ################################
+# Early stopping to stop learning after some criterion 
+early_stopping_params = {'monitor'   : 'val_loss',  # Value to monitor
+                         'min_delta' : 0.0001,          # Minimum delta to declare an improvement
+                         'patience'  : 10,          # How much time to wait for an improvement
+                         'verbose'   : 1,           # Verbosity level
+                         'restore_best_weights':True,
+                         'mode'      : 'min'}       # Mode : 'auto', 'min', 'max'
+
+# Reduce LR on plateau : if no improvement for some time, will reduce lr by a certain factor
+reduceLR_params = {'monitor'    : 'val_loss',   # Value to monitor
+                   'factor'     : 0.1,          # Multiplicative factor by which to multiply LR
+                   'min_lr'     : 1e-6,         # Minimum value for LR
+                   'min_delta'  : 0.0001,       # Minimum delta to declare an improvement
+                   'patience'   : 5,            # How much time to wait for an improvement
+                   'cooldown'   : 1,            # How many epochs before starting again to monitor
+                   'verbose'    : 1,            # Verbosity level
+                   'mode'      : 'min'}         # Mode : 'auto', 'min', 'max'
+    
+#################################  Scan dictionary   ##################################
+# /!\ Lists must always contain something (even if 0), otherwise 0 hyperparameters #
+# Classification #
+grouped_loss = GroupedXEnt(group_ids)
+#p = { 
+#    'lr' : [0.01,0.001], 
+#    'first_neuron' : [64,128,256,512,1024],
+#    'activation' : [relu],
+#    'dropout' : [0.,0.1,0.2],
+#    'hidden_layers' : [2,3,4,5,6], # does not take into account the first layer
+#    'output_activation' : [softmax],
+#    'l2' : [0,0.01,0.1],
+#    'optimizer' : [Adam],  
+#    'epochs' : [50],   
+#    'batch_size' : [10000], 
+#    'n_particles' : [16],
+#    'loss_function' : [grouped_loss] , #  [categorical_crossentropy]
+#}
+p = { 
+<<<<<<< HEAD:MachineLearning/HHMachineLearning/parameters_boosted.py
+    'lr' : [0.1], 
+=======
+    'lr' : [0.001], 
+>>>>>>> fd35d2cb6e4f979520e37c17eeacb7ff1354f9dd:MachineLearning/HHMachineLearning/parameters.py
```

Comment:
same

Reply:
I kept basically 2 files there.
`parameters.py` : as used in resolved 
`parameters_boosted.py` : for boosted case